### PR TITLE
add NvidiaSensors widget

### PR DESCRIPTION
**Pull request by akkaky**
Simple widget for display nvidia GPU sensors data.

### CODE REVIEW

**Comment by akkaky** [comment]
`make lint` drop with next error:
```
mypy run-test: commands[1] | mypy -p libqtile
libqtile/widget/pulse_volume.py:4: error: Module 'libqtile.widget._pulse_audio' has no attribute 'ffi'
libqtile/widget/pulse_volume.py:4: error: Module 'libqtile.widget._pulse_audio' has no attribute 'lib'
Found 2 errors in 1 file (checked 141 source files)
ERROR: InvocationError for command /home/akkaky/projects/qtile/.tox/mypy/bin/mypy -p libqtile (exited with code 1)```

What have I fix? I did not touch this file.

**Comment by elParaguayo** [comment]
That error is different to the one causing the checks to fail here. Your failed build check here is a PEP8 one: https://github.com/qtile/qtile/pull/2298/checks?check_run_id=2107757148#step:5:1502

Looks like it wants you to remove one of the blank lines after your last import line.

I'll let one of the devs comment on the mypy/_pulse_audio error.

**Comment by akkaky** [comment]
@elParaguayo, thanks for the help!

File: libqtile/widget/nvidia_sensors.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+from libqtile.widget import base
+
+sensors_mapping = {
+    'fan_speed': 'fan.speed',
+    'perf': 'pstate',
+    'temp': 'temperature.gpu',
+}
+
+
+class NvidiaSensors(base.InLoopPollText):
```
Comment (tych0) [inline libqtile/widget/nvidia_sensors.py:13]: Might be worth making this part of the threadpool, as doing it synchronously will hang the WM if the command ever hangs.

**Comment by akkaky** [comment]
Changed to threadpoll class.

**Comment by tych0** [comment]
Might be nice to add a test to the regex stuff to make sure the regexes keep working if someone refactors them, but LGTM, thanks!

**Pull request merged**